### PR TITLE
Selenium: stabilize FindUsagesBaseOperationTest selenium test

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR reworks use steps in **FindUsagesBaseOperationTest** selenium test in part that checks working with the "**Find Usages**" panel by clicks. Also were added timeout after every click on node.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/11171

### CODE REVIEW

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindUsages.java [hunk 1]
```diff
@@ -109,6 +111,7 @@ public String getTextFromFindUsagesPanel() {
   public void clickOnIconNodeInFindUsagesPanel(String nameNode) {
     seleniumWebDriverHelper.waitAndClick(
         By.xpath(format(Locators.FIND_USAGES_ICON_NODE, nameNode)));
+    WaitUtils.sleepQuietly(500, TimeUnit.MILLISECONDS);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindUsages.java]: Could you, please, leave a comment in the code with explanation why do we need to wait here for 500 millisecs?

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindUsages.java [hunk 2]
```diff
@@ -119,6 +122,7 @@ public void clickOnIconNodeInFindUsagesPanel(String nameNode) {
   public void selectNodeInFindUsagesByDoubleClick(String node) {
     seleniumWebDriverHelper.moveCursorToAndDoubleClick(
         By.xpath(format(Locators.FIND_USAGES_NODE, node)));
+    WaitUtils.sleepQuietly(500, TimeUnit.MILLISECONDS);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindUsages.java]: Could you, please, leave a comment in the code with explanation why do we need to wait here for 500 millisecs?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindUsagesBaseOperationTest.java [hunk 3]
```diff
@@ -67,85 +67,83 @@
...
 
   @Test
   public void checkFindUsagesBaseOperation() {
-    ide.waitOpenedWorkspaceIsReadyToUse();
     projectExplorer.waitItem(PROJECT_NAME);
     projectExplorer.quickExpandWithJavaScript();
     projectExplorer.openItemByVisibleNameInExplorer("AppController.java");
 
-    // Check basic operations of the 'find usages' panel
+    // Check basic operations of the FindUsages panel
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindUsagesBaseOperationTest.java]: `FindUsages` -> `Find Usages`

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindUsagesBaseOperationTest.java [hunk 4]
```diff
@@ -67,85 +67,83 @@
...
+    findUsages.waitFindUsagesPanelIsClosed();
+
+    // Switch to the FindUsages panel and check its expected content
+    findUsages.clickFindUsagesIcon();
     findUsages.waitFindUsagesPanelIsOpen();
     findUsages.waitExpectedTextInFindUsagesPanel(EXPECTED_TEXT);
     findUsages.waitSelectedElementInFindUsagesPanel("numGuessByUser");
 
-    // Check nodes in the 'find usages' panel by 'double click' and click on the icon node
+    // Check nodes in the FindUsages panel by 'double click' and click on the icon node
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindUsagesBaseOperationTest.java]: Code will be slightly clearer if we put comment `click on the icon node` at the proper place in the code below.

**Review by dmytro-ndp — APPROVED** [review]

**Review by artaleks9 — APPROVED** [review]

**Pull request merged**